docs/upgrade.md

```diff
@@ -119,17 +130,15 @@ The new `capabilities`
 
 are now active by default.
 
-# Upgrading to v1.53.0
-
-## Dropping support for `webclient` listeners and non-HTTP(S) `web_client_location`
+## Removal of `user_may_create_room_with_invites`
 
-Per the deprecation notice in Synapse v1.51.0, listeners of type  `webclient`
-are no longer supported and configuring them is a now a configuration error.
+As announced with the release of [Synapse 1.47.0](https://matrix-org.github.io/synapse/latest/upgrade#deprecation-of-the-user_may_create_room_with_invites-module-callback),
```

Comment:
It might make sense to use just the fragment here so if you're viewing it on GitHub or the website it works fine.

```suggestion
As announced with the release of [Synapse 1.47.0](#deprecation-of-the-user_may_create_room_with_invites-module-callback),
```